src/test/java/org/example/Operations/ResourceOperationsTest.java

```diff
@@ -0,0 +1,357 @@
+package org.example.Operations;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.example.Properties.DataObject.DataObjectRemoveParams;
+import org.example.Properties.DataObject.DataObjectWriteParams;
+import org.example.Properties.Resource.ResourceCreateParams;
+import org.example.Serialize.ModifyMetadataOperations;
+import org.example.Util.Response;
+import org.example.Wrapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Timeout;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class ResourceOperationsTest {
+    private Wrapper rods;
+    private String rodsToken;
+
+    private Wrapper alice;
+    private String aliceToken;
+
+    private String host;
+    private final ObjectMapper mapper = new ObjectMapper();
+    private static final Logger logger = LogManager.getLogger(ResourceOperationsTest.class);
+
+    @Before
+    public void setup() {
+        host = "52.91.145.195";
+        String port = "8888";
+        String version = "0.3.0";
+
+        String baseUrl = "http://" + host + ":" + port + "/irods-http-api/" + version;
+
+        // Create client with rodsadmin status
+        rods = new Wrapper(baseUrl, "rods", "rods");
+        rods.authenticate();
+        rodsToken = rods.getAuthToken();
+
+        // Create a client with rodsuser status
+        alice = new Wrapper(baseUrl, "alice", "alicepass");
+        alice.authenticate();
+        aliceToken = alice.getAuthToken();
+    }
+
+    @Test
+    @Timeout(5000)
+    public void testCommonOperations() {
```

Comment:
What is the motivation behind the `@Timeout`?

Reply:
The method that causes the test to fail would take a long time to end, so I had the timeout for that and I forgot to remove it.

Reply:
So the test is good/passing consistently without that annotation?

Reply:
That test is still failing, but the tests will still run and complete without the annotation.

Reply:
It's about this issue number #61

Reply:
Oh right. I think the issue with that has to do with the fact you're running the tests from the host environment and the HTTP API running in a completely different environment. The HTTP API's python tests assume everything is running local to the iRODS server.

Reply:
Basically, you have a hostname issue.

Reply:
Ahh, I see. That makes sense.

Reply:
I've marked the test as ignore for now.